graphql_compiler/schema_generation/sqlalchemy/scalar_type_mapper.py

```diff
@@ -94,8 +96,50 @@
     mssqltypes.XML,
 })
 
-SQL_CLASS_TO_GRAPHQL_TYPE = merge_non_overlapping_dicts(
-    GENERIC_SQL_CLASS_TO_GRAPHQL_TYPE, MSSQL_CLASS_TO_GRAPHQL_TYPE)
+
+POSTGRES_CLASS_TO_GRAPHQL_TYPES = {
+    postgrestypes.UUID: GraphQLID,
+    postgrestypes.DOUBLE_PRECISION: GraphQLFloat,
+    postgrestypes.TIMESTAMP: GraphQLDateTime,
+    postgrestypes.ENUM: GraphQLString,
+}
+
+UNSUPPORTED_POSTGRES_TYPES = frozenset({
+    # We shouldn't map the Postgresql bit type to the GraphQLBoolean type. The Postgresql bit type
+    # is used to represent a bit string of variable length.
+    # https://www.postgresql.org/docs/8.1/datatype-bit.html
+    postgrestypes.TIME,
+    postgrestypes.BIT,
+    postgrestypes.INET,
+    postgrestypes.CIDR,
+    postgrestypes.MACADDR,
+    postgrestypes.MONEY,
+    postgrestypes.OID,
+    postgrestypes.REGCLASS,
+    postgrestypes.BYTEA,
+    postgrestypes.INTERVAL,
+    postgrestypes.ARRAY,
```

Comment:
We don't support array types? What would be required for us to be able to turn them into GraphQL list types?

Reply:
We don't have the functionality to support SQL ARRAYS. We could add it. It would probably be around 30-50 lines of code with tests included. Would you prefer that I did this in this PR or in a separate PR?

Reply:
I prefer separate because I want `DOUBLE_PRECISION` to merge into master sooner